src/webui/www/public/scripts/dynamicTable.js

```diff
@@ -803,10 +804,36 @@ var TorrentsTable = new Class({
                 else
                     td.adopt(new Element('img', {
                         'src' : img_path,
-                        'class' : 'statusIcon'
+                        'class' : 'stateIcon'
                     }));
             };
 
+            // status
+
+            this.columns['status'].updateTd = function (td, row) {
+                var status = this.getRowValue(row);
+
+                if (status === "downloading" || status === "forcedDL" || status === "metaDL")
```

Comment:
we would require adding parentheses for every `==` operator, but I'm not sure if that rule also applies to JavaScript...